celavi/pylca_celavi/pylca_celavi_background_postprocess.py

```diff
@@ -1,3 +1,9 @@
+# TODO: Add a short module docstring above the code to:
+#  1) provide authors, date of creation
+#  2) give a high level description (2-3 lines) of what the module does
+#  3) write any other relevant information
+
+# TODO: if sys, pickle, and os are not used, consider removing
```

Comment:
Leaving this one for TJ to review in detail

Reply:
Completed in PR #101